TapOrHoldCounter/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,83 @@
+                        <viewControllerLayoutGuide type="top" id="y3c-jy-aDJ"/>
+                        <viewControllerLayoutGuide type="bottom" id="wfy-db-euE"/>
+                    </layoutGuides>
+                    <view key="view" contentMode="scaleToFill" id="8bC-Xf-vdC">
+                        <rect key="frame" x="0.0" y="0.0" width="600" height="600"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="0" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="udO-OA-p90">
+                                <rect key="frame" x="284" y="269" width="31" height="60"/>
+                                <fontDescription key="fontDescription" type="system" pointSize="50"/>
+                                <color key="textColor" cocoaTouchSystemColor="darkTextColor"/>
+                                <nil key="highlightedColor"/>
+                            </label>
+                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="YoR-Sl-yZL">
+                                <rect key="frame" x="260.5" y="329.5" width="80" height="30"/>
+                                <state key="normal" title="Tap or Hold"/>
+                                <connections>
+                                    <action selector="onTapOrHold:" destination="BYZ-38-t0r" eventType="touchUpInside" id="HIe-O6-pN7"/>
+                                    <action selector="onTapOrHold:" destination="BYZ-38-t0r" eventType="touchDragInside" id="ZzS-fA-b02"/>
+                                </connections>
+                            </button>
+                        </subviews>
+                        <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="calibratedWhite"/>
+                        <constraints>
+                            <constraint firstItem="udO-OA-p90" firstAttribute="centerY" secondItem="8bC-Xf-vdC" secondAttribute="centerY" constant="-1" id="5tt-MN-5xk"/>
+                            <constraint firstItem="YoR-Sl-yZL" firstAttribute="top" secondItem="udO-OA-p90" secondAttribute="bottom" constant="8" symbolic="YES" id="TBH-i3-57C"/>
+                            <constraint firstItem="YoR-Sl-yZL" firstAttribute="centerX" secondItem="8bC-Xf-vdC" secondAttribute="centerX" id="e8i-eS-Esq"/>
+                            <constraint firstItem="udO-OA-p90" firstAttribute="centerX" secondItem="8bC-Xf-vdC" secondAttribute="centerX" id="sqK-Ou-TtQ"/>
```

Comment:
On the storyboard I saw a constraint that is repeated (it looks like disabled). Maybe one of these?